main/src/library/Array.flix

```diff
@@ -1150,6 +1150,12 @@ namespace Array {
     pub def toMap(a: Array[(a, b)]): Map[a, b] & Impure =
         foldRight((x, m) -> Map.insert(fst(x), snd(x), m), Map.empty(), a)
 
+    ///
+    /// Returns the array `a` as a MutList.
+    ///
+    pub def toMutList(a: Array[a]): MutList[a] & Impure =
```

Comment:
I think it would also be nice to have a `List.toMut`.

Reply:
I agree.